controllers/admin/index.go

```diff
@@ -39,9 +39,12 @@ func ServeAdmin(w http.ResponseWriter, r *http.Request) {
 
 	mimeType := mime.TypeByExtension(filepath.Ext(path))
 	w.Header().Set("Content-Type", mimeType)
-	w.Write(b)
+	_, err = w.Write(b)
+	if err != nil {
```

Comment:
I don't know how others feel, but I personally prefer to write these type of two lines as one. So:

```go
if _, err = w.Write(b); err != nil {
```

Reply:
Agreed.  Too bad there wasn't a linter rule for that!  I won't go back the code base and change them all but we should standardize on that going forward.